src/layout/base/TopNavigation.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { PropsWithChildren } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { MdOutlineOpenInNew } from 'react-icons/md';
+import { GitHubButton, MastodonButton, TwitterButton } from './SocialIcons';
+
+export function TopNavigation(props: { hideInternalLinks: boolean | undefined }) {
+  return (
+    <nav id='right-nav-buttons' className='items-center justify-center '>
+      <ul className='ml-1 flex items-center gap-6 xl:gap-8'>
+        <li>
+          <NavigationLink path={'https://genspectrum.org'}>
+            <span className='inline-flex gap-1'>
```

Comment:
```suggestion
            <span className='inline-flex items-center gap-1'>
```
Before:
![grafik](https://github.com/user-attachments/assets/de37a4c2-a508-4ac1-9576-ef15a91bf8d8)


After:
![grafik](https://github.com/user-attachments/assets/03f9d9c4-10bb-4981-9588-5509de251b17)